ui/src/app/components/workspace/workspace.component.ts

```diff
@@ -35,6 +36,9 @@ export class WorkspaceComponent implements OnInit, OnDestroy {
   spannerTree: ISchemaObjectNode[] = []
   srcTree: ISchemaObjectNode[] = []
   issuesAndSuggestionsLabel: string = 'ISSUES AND SUGGESTIONS'
+  objectExplorerInitiallyRender: boolean = false
+  srcDbName: string = localStorage.getItem(StorageKeys.SourceDbName) as string
+  conversionRatePercentages: number[] = [0, 0, 0]
```

Comment:
better to have this as an object instead of array, it will be easier to understand

Reply:
Changed to object